style.css

```diff
@@ -1,24 +1,10 @@
-.container {  display: grid;
-    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
-    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+.container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(4, 1fr);
     gap: 0px 0px;
     grid-auto-flow: row;
-    grid-template-areas:
-      ". . . . . ."
-      "core-1 core-2 core-3 . . ."
-      "lab-1 lab-2 lab-3 lab-4 lab-5 lab-6"
-      "type-1 type-2 type-3 . . ."
-      ". . . . . ."
-      ". . . . . .";
-  }
-  
-  .core-1 { grid-area: core-1;}
-  
-  .core-2 { grid-area: core-2; }
-  
-  .core-3 { grid-area: core-3; }
-  
-  .core-4 { grid-area: core-4; }
-  
-  .core-5 { grid-area: core-5; }
-  
```

Comment:
No need to assign template areas, the idea behind grid is that you define a grid and the browser populates it for you